### PR TITLE
Modernize: introduce kehaar, upgrade langohr & resource-config

### PR DESCRIPTION
**Pull request by tie-rack**
Bring this up to date with how we're building services now.

(supersedes #6)

### CODE REVIEW

File: src/sms_works/core.clj [hunk 1]
```diff
@@ -1,52 +1,13 @@
 (ns sms-works.core
-  (:require [clojure.edn :as edn]
-            [clojure.tools.logging :as log]
-            [twilio.core :as twilio]
+  (:require [clojure.tools.logging :as log]
             [turbovote.resource-config :refer [config]]
-            [langohr.consumers :as lc]
             [sms-works.queue :as queue]
-            [clojure.string :as str])
+            [immutant.util :as immutant])
```
Comment (ddellacosta) [inline src/sms_works/core.clj]: Why include immutant if we are getting rid of Wildfly across the system--can't we just do something like what Wes suggested [here](https://github.com/democracyworks/turbovote-admin-http-api/pull/15#discussion_r94177797)?

Comment (tie-rack) [inline src/sms_works/core.clj]: Yeah, even though we got rid of wildfly, immutant is still going strong in all the other repos. I'll try that approach.

**Comment by tie-rack** [comment]
@ddellacosta `.addShutdownHook` works!

**Comment by tie-rack** [comment]
I think changing the queue name is orthogonal to upgrading. So I'd like to get this out separately. Ready for re-review @ddellacosta

**Pull request merged**